packages/kit/src/vite/build/build_service_worker.js

```diff
@@ -55,9 +55,7 @@ export async function build_service_worker(
 			];
 
 			export const prerendered = [
-				${prerendered.paths
-					.map((path) => s(normalize_path(path, config.kit.trailingSlash)))
-					.join(',\n\t\t\t\t')}
+				${prerendered.paths.map((path) => s(path)).join(',\n\t\t\t\t')}
```

Comment:
Just to clarify: We don't normalize here because we already did in `prerender.js`?

Reply:
exactly, yep. previously this was normalising pages _and_ endpoints; we only want pages to be normalised, hence moving the logic from here to there